tests/zfs-tests/tests/functional/cli_root/zpool_expand/zpool_expand_002_pos.ksh

```diff
@@ -83,10 +84,33 @@ for type in " " mirror raidz raidz2; do
 	typeset zfs_prev_size=$(zfs get -p avail $TESTPOOL1 | tail -1 | \
```

Comment:
I know it's not your change, but this could be `typeset zfs_prev_size=$(get_prop avail $TESTPOOL1)`

Reply:
Sure - just changed it.